src/clients/createClient.ts

```diff
@@ -222,6 +231,7 @@ export function createClient(parameters: ClientConfig): Client {
     name = 'Base Client',
     pollingInterval = 4_000,
     type = 'base',
+    mode = undefined,
```

Comment:
```suggestion
    mode,
```
Not sure why I did the default